src/icons/IconWarningHex.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export function IconWarningHex() {
```

Comment:
@steiningermona since we are migrating all the icons.. please make this to accept a size property that will be ignored in this version, but will be used on coming icon versions.